src/main/java/org/testeditor/fixture/commons/date/generate/DateGenerator.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.commons.date.generate;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This Class is for convenience purposes to use in tests. To create Birthday dates in the past with a negative number
+ * as offset or a positive number for birthday dates in the future. <p>
+ * 
+ * The method generateDateFromNowWithOffset() returns a date as String in a given format like e.g.
+ * "dd.MM.yyyy" with a given offset in days henceforth. <br>
+ * E.g. current date = 02.08.2014; offset = 14; <br>
+ * You will get a date as String like "16.08.2014". <br>
+ * For a negative offset = -14 and current date = 02.08.2014 <br>
+ * You will get a date as String like "19.07.2014".
+ */
+class DateGenerator {
+    
+    /**
+     * This method returns a date as String in a given format like e.g.
+     * "dd.MM.yyyy" with a given offset in days henceforth. <br>
+     * E.g. current date = 02.08.2014; offset = 14; <br>
+     * You will get a date as String like "16.08.2014". <br>
+     * For a negative offset = -14 and current date = 02.08.2014 <br>
+     * You will get a date as String like "19.07.2014".
+     *
+     * @param daysToAdd
```

Comment:
since it can be a negative number, how about calling it `offsetInDays` or something?